instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -95,6 +102,11 @@ def process_request(self, request):
         tracer = get_tracer(__name__, __version__)
 
         attributes = collect_request_attributes(environ)
+        for attr in self._traced_request_attrs:
+            if hasattr(request, attr):
+                value = str(getattr(request, attr))
+                if value:
+                    attributes[attr] = value
```

Comment:
If an attr is `None`, you would get `value = 'None'`. Is this intended?

Reply:
I changed it to ignore when value is set to None.